Disable -march=native in the code, as this is not available on ppc64le and -mcpu=native is in the CFLAGS and CXXFLAGS
--- vendor/thread_pool/CMakeLists.txt.orig	2020-02-11 16:16:27.402694000 +0000
+++ vendor/thread_pool/CMakeLists.txt	2020-02-11 16:16:36.861745000 +0000
@@ -7,7 +7,7 @@
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic -march=native")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic")
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
--- vendor/spoa/CMakeLists.txt.orig	2020-02-11 16:19:35.620937000 +0000
+++ vendor/spoa/CMakeLists.txt	2020-02-11 16:19:47.134503000 +0000
@@ -14,7 +14,7 @@
 
 option(spoa_build_executable "Build spoa standalone tool" OFF)
 option(spoa_build_tests "Build spoa unit tests" OFF)
-option(spoa_optimize_for_native "Buiold spoa with march=native" ON)
+option(spoa_optimize_for_native "Buiold spoa with march=native" OFF)
 
 if (spoa_optimize_for_native)
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -march=native")
--- vendor/ClaraGenomicsAnalysis/cudaaligner/CMakeLists.txt.orig	2020-02-11 16:25:23.940669000 +0000
+++ vendor/ClaraGenomicsAnalysis/cudaaligner/CMakeLists.txt	2020-02-11 16:25:46.566066000 +0000
@@ -39,7 +39,7 @@
 
 target_link_libraries(cudaaligner utils logging)
 
-target_compile_options(cudaaligner PRIVATE -Werror -Wall -Wextra -march=native)
+target_compile_options(cudaaligner PRIVATE -Werror -Wall -Wextra)
 
 target_include_directories(cudaaligner 
     PUBLIC 
--- vendor/ClaraGenomicsAnalysis/cudapoa/CMakeLists.txt.orig	2020-02-11 16:26:24.683320000 +0000
+++ vendor/ClaraGenomicsAnalysis/cudapoa/CMakeLists.txt	2020-02-11 16:26:35.297104000 +0000
@@ -27,7 +27,7 @@
 
 target_link_libraries(cudapoa utils logging)
 
-target_compile_options(cudapoa PRIVATE -Werror -Wall -Wextra -march=native)
+target_compile_options(cudapoa PRIVATE -Werror -Wall -Wextra)
 
 if(cga_enable_cudapoa_nw_print)
     target_compile_definitions(cudapoa PUBLIC NW_VERBOSE_PRINT)
